db/seeds.rb

```diff
@@ -10,5 +10,9 @@
 Restaurant.destroy_all
 
 FactoryBot.create_list(:user, 2)
-FactoryBot.create_list(:restaurant, 2, active: true)
 FactoryBot.create_list(:restaurant, 2, active: false)
+
+FactoryBot.create_list(:restaurant, 2, active: true) do |restaurant|
```

Comment:
where is each? :)

Reply:
because as I remember this method returns an array and to go through the array we need `each`

Reply:
Add, but it worked fine without each))))